applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -417,6 +421,24 @@ def ExecuteInitializeSolutionStep(self):
                                 self.step = 0  # Reset
                                 self.time = 0.0  # Reset
 
+    def ExecuteFinalize(self):
+        """ This method is executed in order to finalize the simulation and save the refined mesh in a new .mdpa file
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        """
+        self.remesh_at_finalize = self.settings["remesh_at_finalize"].GetBool()
```

Comment:
Why do you need to save it as a member var if you are only using it within this function scope?